hardhat/test/unit/21_Utils.t.js

```diff
@@ -0,0 +1,122 @@
+const {
+  loadFixture,
+} = require("@nomicfoundation/hardhat-toolbox/network-helpers");
+const { expect } = require("chai");
+const hre = require("hardhat");
+const { ethers } = hre;
+const { fixtures } = require("../helper_fixture");
+
+// describe.only("Utils", function () {
+describe("Utils", function () {
+  const mockedTypes = [
+    "address",
+    "address",
+    "uint",
+    "uint",
+    "string",
+    "uint",
+    "uint",
+  ];
+  const mockedParams = [
+    // owner.address,
+    // user.address,
```

Comment:
remove commented code

Reply:
Nope. This comment is there to help the reader of the test by knowing the structure of the data